Config/DefaultInput.ini

```diff
@@ -75,6 +75,8 @@ DefaultViewportMouseLockMode=LockOnCapture
 FOVScale=0.011110
 DoubleClickTime=0.200000
 +ActionMappings=(ActionName="Interact",bShift=False,bCtrl=False,bAlt=False,bCmd=False,Key=SpaceBar)
++ActionMappings=(ActionName="Plant",bShift=False,bCtrl=False,bAlt=False,bCmd=False,Key=E)
```

Comment:
Should be named "Interact" as this action will do more than just plant in later versions of the game.